styles.css

```diff
@@ -91,3 +142,153 @@ body {
 .headline-text {
   letter-spacing: -0.01em;
 }
+
+/* ==================== Works styles ==================== */
+
+.works {
+  width: 100%;
+  max-width: 500px;
+  align-items: center;
+}
+
+.works h1 {
+  padding: 40px 32px 32px 16px;
+  width: 100%;
+}
+
+.projects {
+  display: grid;
+  width: 100%;
+  justify-content: center;
+  text-align: left;
+  grid-template-columns: 100%;
+}
+
+.project {
+  height: 342px;
+  list-style: none;
+}
```

Comment:
I also did that, but with a different class name.
it was my last review idea to avoid repetition I created two classes for flexbox :

<img width="236" alt="image" src="https://user-images.githubusercontent.com/77485409/228843005-9f98738d-f2b0-4080-925a-860f41fe7cd0.png">
<img width="300" alt="image" src="https://user-images.githubusercontent.com/77485409/228843771-15d62345-5dfd-4352-97d2-0dde28cafc2f.png">